src/main/java/uk/gov/pay/api/validation/SearchValidator.java

```diff
@@ -5,17 +5,29 @@
 import static org.eclipse.jetty.util.StringUtil.isNotBlank;
 
 
-public abstract class SearchValidator {
```

Comment:
Not sure why `abstract` was added, is it because we want to make sure that we can't initialise this class directly and we need specific implementations? On the other hand all the methods are static there so we will not use it as an object anyway

Reply:
I think it was abstract because originally the refund class was extending this, but being it a collection of static methods there was no need to do so. It is not abstract anymore